examples/time_series/AR.myst.md

```diff
@@ -81,41 +88,39 @@ slideshow:
 ---
 with pm.Model() as ar1:
     # assumes 95% of prob mass is between -2 and 2
-    theta = pm.Normal("theta", 0.0, 1.0)
+    rho = pm.Normal("rho", mu=0.0, sigma=1.0, shape=2)
```

Comment:
why does this need shape=2 for AR1?

Reply:
Because it has a constant term? Before the observations had it symbolically subtracted `observed = y - center` which is no longer allowed

Reply:
Correct. Note the `constant=True` argument, which adds an intercept term.